scripts/globals/hunts.lua

```diff
@@ -1586,19 +1586,19 @@ if huntEntry then
     local huntZone = player:getCharVar("[hunt]zone")
     local huntId = player:getCharVar("[hunt]id")
     local scyldBounty = zone[huntZone].hunt[huntId].bounty
-
-    player:messageSpecial(msg[5])
-    player:messageSpecial(msg[6], scyldBounty, scylds)
-    player:addCurrency("scyld", scyldBounty)
     -- give player evoliths here
     player:setCharVar("[hunt]nextHunt", getVanaMidnight())
     tpz.hunts.clearHuntVars(player)
 
     -- scylds cap at 1000
-    if player:getCurrency("scyld") + bounty > 1000 then
+    if player:getCurrency("scyld") + scyldBounty > 1000 then
        player:setCurrency("scyld", 1000)
+       player:messageSpecial(msg[5])
+       player:messageSpecial(msg[6], scyldBounty, scylds)
     else
-       player:addCurrency("scyld", bounty)
+       player:addCurrency("scyld", scyldBounty)
+       player:messageSpecial(msg[5])
+       player:messageSpecial(msg[6], scyldBounty, scylds)
```

Comment:
This will still give the old total, go with this:
```lua
    -- scylds cap at 1000
    if player:getCurrency("scyld") + scyldBounty > 1000 then
       player:setCurrency("scyld", 1000)
    else
       player:addCurrency("scyld", scyldBounty)
    end
    player:messageSpecial(msg[5])
    player:messageSpecial(msg[6], scyldBounty, player:getCurrency("scyld"))
```